src/main/java/com/google/api/codegen/csharp/CSharpGapicContext.java

```diff
@@ -633,6 +662,19 @@ private String getTypeName(ProtoElement elem) {
             }
           }
         });
+    if (isPageStreaming) {
+      String[] pageToken = {
+          "/// <param name=\"pageToken\">The token returned from the previous request.",
+          "/// <c>Null</c> or an empty string retrieves the first page.</param>",
+      };
+      String[] pageSize = {
+          "/// <param name=\"pageSize\">The size of page to request, the response will not be larger",
+          "/// than this, but may be smaller. <c>Null</c> or 0 uses a server-defined page size.</param>",
```

Comment:
I'd change to 

A value of <c>null</c> or 0 uses a [...]

That's easier to read and avoids null having a capital N.